miden-lib/asm/kernels/transaction/main.masm

```diff
@@ -64,13 +64,25 @@ proc.main.1
     # Prologue
     # ---------------------------------------------------------------------------------------------
 
+    # add redundant instructions to make span non-empty
+    push.0 drop
+    trace.131072
```

Comment:
Can we use constants with `trace` instruction? If we can, I'd rather define constants at the top of the file and then use them here.